lisa/wa.py

```diff
@@ -33,6 +33,8 @@
 from lisa.git import find_shortest_symref, get_commit_message
 from lisa.trace import Trace
 
+# for Jankbench Results Import
+import sqlite3
```

Comment:
Done!

Reply:
Actually I got confused, `sqlite3` is part of the standard library so it would go in the first group of imports, not the 2nd one, sorry about the churn :(